build.sbt

```diff
@@ -80,3 +94,21 @@ fork in Test := true
 
 // forward stdin/out to fork, so the OpenIE CLI can be run in sbt.
 connectInput in run := true
+
+// The OpenIE codebase has many insignificant style warnings that appear
+// whenever the code is compiled. There are also some important-sounding
+// warnings which don't seem to affect the behavior but should be
+// fixed eventually:
+//
+//   Classes must override both equals and hashCode, not just one
+//   Line is more than 100 characters long
+//   Tab in Scala file
+//   Unusual class name; consider renaming
+//   illegal.imports.message
```

Comment:
@aimichal : The *style* warnings are totally fine to disable (and, generally, ignore). I'd leave the `styleCheck` override in place - it's fairly harmless - but have a simpler comment:
```scala
// OpenIE does not conform to AI2 style guildelines (yet). Disable for now.
// TODO(michalg): Fix to conform.
(StylePlugin.StyleKeys.styleCheck in Compile) := {
  org.scalastyle.OutputResult(0, 0, 0, 0)
}
```
You could also open an issue to update the style.

The compiler warnings are a mixture of legitimate errors (hash code / equals being out-of-sync) and legitimate warnings that may eventually turn into errors (deprecation notices, advanced language features). These should either be fixed, or lived with. They inform our users of current and future *breakages*, not just bad style.